deployer/utils.py

```diff
@@ -46,4 +46,33 @@ def decrypt_file(encrypted_path):
             '--output', f.name,
             '--decrypt', encrypted_path
         ])
-        yield f.name
\ No newline at end of file
+        yield f.name
+
+def add_staff(staff, user_list):
+    """
+    Add the staff ids to the existing user list
+    """
+    if isinstance(user_list, str):
+        user_list = [user_list]
+
+    custom_users = user_list[:]
+    for staff_list_type, staff_ids in staff.items():
+        staff_placeholder = "<staff_" + staff_list_type + ">"
+        if staff_placeholder in user_list:
+            custom_users.remove(staff_placeholder)
+
+            return custom_users + staff_ids
+
+def update_authenticator_config(config):
+    """Prepare a hub's configuration file for deployment."""
+    # Load the staff config file
+    with open('config/hubs/staff.yaml') as f:
+        staff = yaml.load(f)
+
+    authenticator = config.get("jupyterhub", {}).get("hub", {}).get("config", {}).get("Authenticator", {})
+
+    # `Allowed_users` list doesn't exist for hubs where everyone is allowed to login
+    if authenticator.get("allowed_users", None):
+        authenticator["allowed_users"] = add_staff(staff["staff"], authenticator["allowed_users"])
```

Comment:
If "allowed_users" does not exist, then `authenticator["allowed_users"]` would be `None` as per line 75, right?
How the `add_staff` function will handle that `None`?

Reply:
I think that if `allowed_users` is `None`, then the if-statement won't be triggered and it will just be skipped

Reply:
> Feel free to say that my thoughts do not make any sense 😜 .

It is 2 am now so that would make sense 😉 💤